static/js/components/dashboard/CourseSubRow.js

```diff
@@ -62,7 +62,10 @@ export default class CourseSubRow extends React.Component {
   );
 
   courseRunStatus = (courseRun: CourseRun) => {
-    if (courseRun.status === STATUS_NOT_PASSED) {
+    const { hasFinancialAid } = this.props;
+    if (hasFinancialAid) {
```

Comment:
should this be `hasFinancialAid && courseRun.status === STATUS_PASSED`? Just wondering if we want to display the 'Final grade coming soon' message if the user hasn't passed the course yet.